examples/riddle-prep-buddy-v2/[Release] Riddle Prep App Server.ipynb

```diff
@@ -0,0 +1,354 @@
+      },
+      "outputs": [],
+      "source": [
+        "# TTS-Related Imports\n",
+        "import IPython\n",
+        "import tempfile\n",
+        "import subprocess\n",
+        "import time\n",
+        "from TTS.tts.configs.vits_config import VitsConfig\n",
+        "from TTS.tts.models.vits import Vits\n",
+        "from TTS.utils.audio.numpy_transforms import save_wav\n",
+        "import numpy as np\n",
+        "\n",
+        "\n",
+        "# STT-Related Imports\n",
+        "import io\n",
+        "import wave\n",
+        "import numpy as np\n",
+        "import whisper\n",
+        "import jiwer\n",
+        "import time\n",
+        "import pandas as pd\n",
+        "from tabulate import tabulate\n",
+        "from pydub import AudioSegment\n",
+        "import os\n",
+        "import joblib\n",
+        "import re\n",
+        "from transformers import BertTokenizer, BertModel\n",
+        "import torch\n",
+        "import torch.nn.functional as F\n",
+        "from torch import nn, Tensor\n",
+        "\n",
+        "\n",
+        "# API-Related Imports\n",
+        "from fastapi import FastAPI,Response\n",
+        "from fastapi.middleware.cors import CORSMiddleware\n",
+        "from starlette.middleware.gzip import GZipMiddleware\n",
+        "from fastapi.responses import JSONResponse\n",
+        "from fastapi.responses import StreamingResponse,FileResponse\n",
+        "from fastapi import FastAPI, UploadFile, File\n",
+        "import shutil\n",
+        "from pydantic import BaseModel\n",
+        "from IPython.display import Audio\n",
+        "import uvicorn\n",
+        "import nest_asyncio\n",
+        "from pyngrok import ngrok\n",
+        "import base64\n",
+        "\n",
+        "\n",
+        "nest_asyncio.apply()"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "vfOuc8Pxg8Y_"
+      },
+      "outputs": [],
+      "source": [
+        "from starlette.middleware.gzip import GZipMiddleware\n",
+        "app = FastAPI()\n",
+        "app.add_middleware(\n",
+        "    CORSMiddleware,\n",
+        "    allow_origins=['*'],\n",
+        "    allow_credentials=True,\n",
+        "    allow_methods=['*'],\n",
+        "    allow_headers=['*'],\n",
+        ")\n",
+        "app.add_middleware(GZipMiddleware, minimum_size=50)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "MlS247Nggdnb"
+      },
+      "source": [
+        "## TTS"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "XYUzxcTsbg17"
+      },
+      "outputs": [],
+      "source": [
+        "# Load TTS model\n",
+        "live_config=VitsConfig()\n",
+        "live_config.load_json(\"/content/drive/MyDrive/NSMQ AI Project/Technical/TTS/Prof Elsie Kauffmann/VITS model/vits-elsie/traineroutput/vits_vctk-May-24-2023_11+05PM-23a7a9a3/config.json\")\n",
```

Comment:
@naafidasana this should be moved up the notebook and declared as constants so folks know to change it. Similar to what we did in the all in one notebook